packages/func/src/executors/start/executor.ts

```diff
@@ -1,15 +1,15 @@
-import { StartExecutorSchema } from './schema';
 import { Executor } from '@nrwl/devkit';
-import { build } from '../common/utils';
 import { execSync } from 'child_process';
+import { build } from '../common/utils';
+import { StartExecutorSchema } from './schema';
 
 const executor: Executor<StartExecutorSchema> = async (options, context) => {
   const success = build(context);
 
   if (success) {
     const cwd = context.workspace?.projects[context.projectName].root;
 
-    execSync(`func start --port ${options.port}`, {
```

Comment:
Sorry for ping-ponging this back and forth.
I like this change. Simple and elegant.

However, the `port` property is actually being used by the `init` generators.
When you create more than 1 app, the generator will iterate over the existing projects and increase the port automatically for the new project, so there won't be port conflicts.

Can you bring back the `port` property and add the `additionalFlags` on top of that?

Reply:
Sounds good. No worries, thanks for informing me.